Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="bondziks@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
```

Comment:
The `WORKDIR` should be an absolute path. Please ensure that `/app` is the correct absolute path for your application.